03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+tmp_frames = shots.each_slice(2).to_a
+last_pins = tmp_frames.slice(9..-1)
+last_frame = []
+last_pins.map do |pin|
```

Comment:
```suggestion
last_frame = last_pins.map do |pin|
```

mapメソッドは戻り値を使う前提になりますね。（eachとmapの違いを復習してみてください〜）